test/helpers.coffee

```diff
@@ -11,24 +13,28 @@ else
 Event = require "#{helpers.prefix}server/models/event"
 User  = require "#{helpers.prefix}server/models/user"
 
+userID = null
+
 helpers.before = (done) ->
     @timeout 10000
     start = require "#{helpers.prefix}server"
     start 8888, (err, app, server) =>
         @server = server
         data =
+            docType: 'User'
             email: 'test@cozycloud.cc'
             password: 'password'
             timezone: 'Europe/Paris'
-        User.create data, (err) ->
+        ds.post '/data/', data, (err, response, body) ->
```

Comment:
I think it's because the User model has been removed from the app, because we only need to access the timezone, and there is an API for that in CozyDB.